### PR TITLE
build arg instruction support newline list input

### PR DESCRIPTION
**Pull request by kakaZhou719**
### Describe what this PR does / why we need it

below is my test kubefile: 
```
FROM kubernetes:v1.19.8
ARG value=sealer
ARG MY_ARG=abc
ARG MY_*ARG=abc # not support such key format.will ignoring this key.
ARG ddd= # default value is "".
ARG name=testname \
,PASSWORD1=PASSWORD1,\
aptget=update 
COPY aa .
COPY charts charts
RUN echo ${name}
RUN echo ${MY_ARG}
CMD echo ${PASSWORD1}
```

### CODE REVIEW

File: pkg/parser/parser.go [hunk 1]
```diff
@@ -140,17 +140,21 @@ func dispatchArg(layerValue string, ima *v1.Image) {
...
-	}
-	k := strings.TrimSpace(valueLine[0])
-	if !utils.IsLetterOrNumber(k) {
-		logger.Error("ARG key must be letter or number,invalid ARG format will ignore this key %s.", k)
-		return
+
+	kv := strings.Split(layerValue, ",")
+	for _, element := range kv {
+		valueLine := strings.SplitN(element, "=", 2)
+		if len(valueLine) != 2 {
```
Comment (bxy4543) [inline pkg/parser/parser.go:147]: What if value has "="     
like: runArg="--masters=xxx --nodes=xxx"

Comment (kakaZhou719) [reply pkg/parser/parser.go:147]: if so, key="runArg" and value="--masters=xxx --nodes=xxx"。we only make sure that we got the the right key. the value is user set that is  meaningless for sealer build.

**Review by bxy4543 — APPROVED** [review]

**Comment by fanux** [comment]
Add this feature into docs, and what diff with `ENV`?

**Comment by kakaZhou719** [comment]
> Add this feature into docs, and what diff with `ENV`?

- build `ENV`  instruction: not support at present , it will add later for setting host temp env.
- build `ARG`  instruction: it will merge with `RUN` or `CMD` instruction line and  execute it .

more example usage see : https://github.com/alibaba/sealer/pull/1108

**Pull request merged**